src/scripts/main.js

```diff
@@ -1,3 +1,81 @@
 'use strict';
 
-// write code here
+const tBody = document.querySelector('tbody');
+const addRowButton = document.querySelector('.append-row');
+const removeRowButton = document.querySelector('.remove-row');
+const addColumnButton = document.querySelector('.append-column');
+const removeColumnButton = document.querySelector('.remove-column');
+
+let rowsAmount = tBody.rows.length;
+const tableRows = tBody.rows;
```

Comment:
in this case you can use like this not to repeat yourself
```suggestion
const tableRows = tBody.rows;
let rowsAmount = tableRows.length;
```